boxsdk/session/session.py

```diff
@@ -425,6 +435,28 @@ def _get_retry_request_callable(self, network_response, attempt_number, request)
     def _get_request_headers(self):
         return self._default_headers.copy()
 
+    def _prepare_proxy(self):
+        """
+        Prepares basic authenticated and unauthenticated proxies for requests.
+
+        :return:
+            A prepared proxy dict to send along with the request.
+        :rtype:
+            `dict`
+        """
+        proxy = {}
+        if self._network_config.PROXY_AUTH and self._network_config.PROXY_URL:
+            host = self._network_config.PROXY_URL
+            address = host.split('//')[1]
+            proxy['http'] = \
+                'http://' + self._network_config.PROXY_AUTH.get('user', None) + \
```

Comment:
It might be clearer to use a format string here to show exactly what you're trying to get the resulting string to look like